orchestrators/FinanceOperations/src/DTOs/RuleContext.php

```
@@ -0,0 +1,139 @@
+        string $tenantId,
+        private ?string $budgetId = null,
+        private ?string $amount = null,
+        private ?string $costCenterId = null,
+        private array $costCenterIds = [],
+        private ?string $periodId = null,
+        private ?string $subledgerType = null,
+        private array $transactionTypes = [],
+    ) {
+        $trimmedTenantId = trim($tenantId);
+        if ($trimmedTenantId === '') {
+            throw new \InvalidArgumentException('Tenant ID cannot be empty');
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '29', column '23'). (undefined)

(MissingImport)